src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -64,7 +64,7 @@ const Accordion = ({
   );
 
   const { isMounted } = useMounted(() => {
-    const goToHash = async () => {
+    const goToAndOpenHash = async () => {
```

Comment:
Why rename this? This doesn't actually open anything so it's misleading to call it this.